beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go

```diff
@@ -140,8 +140,17 @@ func (vs *Server) GetBeaconBlock(ctx context.Context, req *ethpb.BlockRequest) (
 		// Set sync aggregate. New in Altair.
 		vs.setSyncAggregate(ctx, sBlk)
 
-		// Set execution data. New in Bellatrix.
-		if err := vs.setExecutionData(ctx, sBlk, head); err != nil {
+		// Get local and builder (if enabled) payloads. Set execution data. New in Bellatrix.
+		localPayload, err := vs.getLocalPayload(ctx, sBlk.Block(), head)
```

Comment:
for deneb will the local payload still be the same of will we get blob bundles here?

Reply:
Correct. Already implemented here: https://github.com/prysmaticlabs/prysm/pull/12454